<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BGCMS</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CMSBrowser" name="ui_cmsBrowser">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>28</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_connectToHost"/>
    <addaction name="action_disconnect"/>
    <addaction name="separator"/>
    <addaction name="action_changeAccount"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_copy"/>
    <addaction name="action_cut"/>
    <addaction name="action_paste"/>
   </widget>
   <widget class="QMenu" name="ui_menu_settings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="ui_menu_plugins">
    <property name="title">
     <string>Plugins</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_edit"/>
   <addaction name="ui_menu_plugins"/>
   <addaction name="ui_menu_settings"/>
  </widget>
  <widget class="QStatusBar" name="ui_statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="ui_tb_main">
   <property name="windowTitle">
    <string>Main</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_rootDirectory"/>
   <addaction name="action_back"/>
   <addaction name="separator"/>
   <addaction name="action_remove"/>
   <addaction name="separator"/>
   <addaction name="action_cut"/>
   <addaction name="action_copy"/>
   <addaction name="action_paste"/>
  </widget>
  <widget class="QToolBar" name="ui_tb_plugins">
   <property name="windowTitle">
    <string>Plugins</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="ui_tb_path">
   <property name="windowTitle">
    <string>Path</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="action_connectToHost">
   <property name="text">
    <string>Connect to host</string>
   </property>
  </action>
  <action name="action_back">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/back.png</normaloff>:/imgs/back.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/new.png</normaloff>:/imgs/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/delete.png</normaloff>:/imgs/delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/copy.png</normaloff>:/imgs/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/paste.png</normaloff>:/imgs/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/cut.png</normaloff>:/imgs/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="action_changeAccount">
   <property name="text">
    <string>Change Account</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_rootDirectory">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/imgs/root.png</normaloff>:/imgs/root.png</iconset>
   </property>
   <property name="text">
    <string>Root Directory</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMSBrowser</class>
   <extends>QTableView</extends>
   <header location="global">cmsbrowser.h</header>
   <slots>
    <signal>atRoot(bool)</signal>
    <slot>back()</slot>
    <slot>newNode()</slot>
    <slot>removeSelectedNodes()</slot>
    <slot>copySelectedNodes()</slot>
    <slot>paste()</slot>
    <slot>cutSelectedNodes()</slot>
    <slot>rootDirectory()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bgcms.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_connectToHost</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>connect2Host()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_disconnect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>disconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_back</sender>
   <signal>triggered()</signal>
   <receiver>ui_cmsBrowser</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_cmsBrowser</sender>
   <signal>atRoot(bool)</signal>
   <receiver>action_back</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_new</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newBlankNode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_remove</sender>
   <signal>triggered()</signal>
   <receiver>ui_cmsBrowser</receiver>
   <slot>removeSelectedNodes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_copy</sender>
   <signal>triggered()</signal>
   <receiver>ui_cmsBrowser</receiver>
   <slot>copySelectedNodes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_paste</sender>
   <signal>triggered()</signal>
   <receiver>ui_cmsBrowser</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_cut</sender>
   <signal>triggered()</signal>
   <receiver>ui_cmsBrowser</receiver>
   <slot>cutSelectedNodes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_changeAccount</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeAccount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_rootDirectory</sender>
   <signal>triggered()</signal>
   <receiver>ui_cmsBrowser</receiver>
   <slot>rootDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connect2Host()</slot>
  <slot>disconnect()</slot>
  <slot>nodeDoubleClicked(QModelIndex)</slot>
  <slot>newBlankNode()</slot>
  <slot>changeAccount()</slot>
 </slots>
</ui>
